sweep/tx_input_set.go

```diff
@@ -384,9 +399,35 @@ func (t *txInputSet) addPositiveYieldInputs(sweepableInputs []*pendingInput) {
 	// We managed to add all inputs to the set.
 }
 
+// AddWalletInputs adds wallet inputs to the set until a non-dust change output
```

Comment:
I think the comment is not 100% correct, I think ` AddWalletInputs` will also succeed if it can pay for fees and the required outputs.

Reply:
you are right - the output can come from the required output or the change output, updated the comments.